File: backend-dotnet-core/backend-dotnet-core/Phonebook.Api/Models/PersonResponse.cs
namespace Phonebook.Api.Models;

public class PersonResponse
{
    public long Id { get; set; }
    public string Name { get; set; } = null!;
    public string Surname { get; set; } = null!;
    public int Age { get; set; }
    public string? Email { get; set; }
    public string PhoneNumber { get; set; } = null!;
    public DateTime CreatedAtUtc { get; set; }
    public DateTime? UpdatedAtUtc { get; set; }
}

File: backend-spring-boot/src/main/java/com/interview/backendspringboot/domain/person/PersonResponse.java
package com.interview.backendspringboot.domain.person;

import java.time.Instant;

public record PersonResponse(
        Long id,
        String name,
        String surname,
        Integer age,
        String email,
        String phone,
        Boolean active,
        Instant createdAt,
        Instant updatedAt
) {}